src/components/settings/SettingsSSOTab/SSOStepper.jsx

```diff
@@ -65,7 +73,8 @@ const SSOStepper = ({ enterpriseSlug, enterpriseId, enterpriseName }) => {
     if (configValues !== null) {
       configValues.append('enterprise_customer_uuid', enterpriseId);
       try {
-        await LmsApiService.updateProviderConfig(configValues, providerConfig.id);
+        const response = await LmsApiService.updateProviderConfig(configValues, providerConfig.id);
```

Comment:
when provider configs are updated, the ID updates. Therefore we need to reset the provider config data whenever we make an update request